docs/001_develop/03_client-capabilities/008_interaction/date-picker.mdx

```diff
@@ -0,0 +1,283 @@
+
+<Tabs defaultValue="genesis" values={[{ label: 'Genesis', value: 'genesis', }, { label: 'React', value: 'react', }, { label: 'Angular', value: 'angular', }]}>
+    <TabItem value="genesis">
+        Declaration
+        ```html
+        <rapid-date-picker></rapid-date-picker>
+        ```
+
+        Usage
+        ```typescript
+        @customElement({
+            name: 'my-element',
+            template: html`
+                <rapid-date-picker @value-changed=${(x, ctx) => x.valueChanged(customEvent(ctx))}>
+                </rapid-date-picker>
+            `,
+        })
+        export class MyElement extends GenesisElement {
+            valueChanged(e: CustomEvent) {
+                console.log('date picker value changed', e.detail);
+            }
+        }
+        ```
+    </TabItem>
+    <TabItem value="react">
+        Declaration
+        ```html
+        <rapid-date-picker></rapid-date-picker>
+        ```
+
+        Usage
+        ```tsx
+        export function MyComponent() {
+
+          const datePickerRef = useRef(null);
+          const [value, setValue] = useState('Not changed');
+          const handleValueChanged = (value) => {
+            setValue(value.detail);
+          }
+
+          useEffect(() => {
+            if (datePickerRef.current) {
+              datePickerRef.current.addEventListener('value-changed', handleValueChanged);
+            }
+          }, []);
+
+          return (
+            <rapid-date-picker ref={datePickerRef}>
+            </rapid-date-picker>
+          );
+        }
+        ```
+    </TabItem>
+    <TabItem value="angular">
+      Usage
+      ```typescript
+      import { AfterViewInit, Component, CUSTOM_ELEMENTS_SCHEMA, ElementRef, ViewChild } from '@angular/core';
+
+      @Component({
+      selector: 'date-picker-example',
+      template: `
+        <rapid-date-picker (value-changed)="handleValueChanged($event)">
+        </rapid-date-picker>
+        `,
+        standalone: true,
+        schemas: [CUSTOM_ELEMENTS_SCHEMA],
+      })
+      export class DatePickerExampleComponent implements AfterViewInit {
+          valueChanged(e) {
+            console.log(e.detail);
+          }
+      }
+      ```
+    </TabItem>
+</Tabs>
+
+## DOM API
+
+The sections below provide property and attribute binding examples for Genesis Component syntax. The HTML closing tag is not included.
+
+### Attributes
+
+<table>
+    <thead>
+        <tr>
+            <th>Attribute</th>
+            <th>Type</th>
+            <th>Use</th>
+            <th>Example</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>format</td>
+            <td><code>string</code></td>
+            <td>The date format shown defaults to <code>MM-DD-YYYY</code> and can accept any <a href="https://day.js.org/docs/en/display/format">valid dayjs format</a></td>
+            <td>
+
+            ```typescript
+            <rapid-date-picker format="DD-MM-YYYY">
+            ```
+
```

Comment:
Real nitpick, but to get the formatting 100% correct you need to left align these cells or markdown renderers such as github won't work
![Screenshot 2025-06-30 at 16 02 09](https://github.com/user-attachments/assets/01373f9f-44e9-4ff1-bd22-141d03e6950d)